certbot-ci/windows_installer_integration_tests/test_main.py

```diff
@@ -56,6 +64,7 @@ def test_it(request):
         pass
 
 
-def _ps(powershell_str, capture_stdout=False):
+def _ps(powershell_str: str, capture_stdout: bool = False) -> Any:
```

Comment:
Is `Any` due to the pain of rewriting `test_it` to type this differently?

Reply:
Yes. I could have duplicated the `_ps` function for the two use cases or make the single function return an `Optional[str]` and check the output in the test function, but I thought it was not worth the added complexity.